mobile/src/bundles/auth/store/slice.ts

```diff
@@ -36,21 +33,26 @@ const { reducer, actions, name } = createSlice({
             (state, { payload }) => {
                 const { email, id, role } = payload;
                 state.dataStatus = DataStatus.FULFILLED;
-                state.userData = { email, role, id, isProfileComplete: false };
+                state.currentUser = {
+                    email,
+                    role,
+                    id,
+                    isProfileComplete: false,
```

Comment:
don't we have last completed step in this data?

Reply:
There will be a **completedStep** field in user-details that will contain the name of the last completed step, or null. 
The name is taken from shared/enum
So, I think I can remove isProfileComplete from here completely. And in the [root navigation](mobile/src/navigations/root/root.tsx), request completedStep through the getTalentDetails action.